src/QMCWaveFunctions/tests/test_multi_slater_determinant.cpp

```diff
@@ -118,6 +121,27 @@ void test_LiH_msd(const std::string& spo_xml_string,
   std::cout << "twf.calcRatio ratio " << ratio << std::endl;
   CHECK(ratio == ValueApprox(1.374307585));
 
+
+  opt_variables_type active;
+  twf.checkInVariables(active);
+
+  int nparam = active.size_of_active();
+  REQUIRE(nparam == 1486);
+
+  using ValueType = QMCTraits::ValueType;
+  std::vector<ValueType> dlogpsi(nparam);
+  std::vector<ValueType> dhpsioverpsi(nparam);
+  twf.evaluateDerivatives(elec_, active, dlogpsi, dhpsioverpsi);
+
+  // Numbers not validated
+  REQUIRE(dlogpsi[0] == ValueApprox(0.006449058893092842));
```

Comment:
https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#natural-expressions
FYI, I learnt that CHECK doesn't stop execution. I found it better suited for value checks.
REQUIRE seems better suited for critical sanity checks.
The above REQUIRE on nparam remains a good choice. If it goes wrong, the execution doesn't need to continue.

Reply:
Useful - most all of our tests could be CHECKs